### ISSUE TITLE
[BUG] Mark as archived/broken more ESLint packages

### ISSUE THREAD

**Issue by AlexWayfer** [pulsar-edit/package-backend#257]
[`fast-eslint`](https://web.pulsar-edit.dev/packages/fast-eslint) is archived and non-working (sees no configuration while it's here, just outdated config formats): https://github.com/arnaud-dezandee/fast-eslint

[`eslint`](https://web.pulsar-edit.dev/packages/eslint) is broken: error after installation, last update was 10 years ago: https://github.com/iancmyers/atom-eslint

This issue is a successor of #244, so we have no ESLint integration with Pulsar Editor right now.

I saw [`linter-eslint-node` fork](https://github.com/savetheclocktower/linter-eslint-node) by @savetheclocktower, but it's not in maintenance.

**Comment by savetheclocktower** [comment]
I still use `linter-eslint-node` (installed directly from the repo, I think) and my feeling is that it'll eventually move to Pulsar Cooperative and live in that repo. (Not looking forward to changing the name.) Try it out and let me know if it doesn't work for you.

But, yeah, the other packages listed are worth marking as broken.

**Comment by AlexWayfer** [comment]
Yeah, I've installed `linter-eslint-node` from official repos and it works, even with the newest versions of ESLint and its config.

Probably, we should update [the package page](https://web.pulsar-edit.dev/packages/linter-eslint-node) and remove "Archived" badge.

**Comment by confused-Techie** [comment]
@AlexWayfer But the repository where the code lives is still archived. That's all the badge is intended to indicate, so we will probably want to go ahead and leave it, even if the package is still working

**Comment by AlexWayfer** [comment]
> @AlexWayfer But the repository where the code lives is still archived. That's all the badge is intended to indicate, so we will probably want to go ahead and leave it, even if the package is still working

Yeah, I mean the fork, which you've mention, probably should replace this package, or be added as a new one into official repos, without this badge.
